src/qililab/calibration/calibration_node.py

```diff
@@ -491,6 +491,8 @@ def _build_notebooks_logger_stream() -> StringIO:
 
     def _execute_notebook(self, input_path: str, output_path: str, parameters: dict | None = None) -> dict:
         """Executes a Jupyter Notebook overwriting the `parameters` cell, and capturing the execution ``output``.
+        The execution of the notebook will happen in the notebook folder. In case the workflow is not beeing executed in the notebook folder,
```

Comment:
Great docstring, I asked copilot for making it shorter and got back:
`This method changes the working directory to the notebook folder before executing the notebook and restores the original working directory after execution if necessary.`

Reply:
Also add a white line before this sentence and the previous 🙌

Reply:
I add it here in case you like it:
```suggestion

        This method changes the working directory to the notebook folder before executing the notebook and restores the original working directory after execution if necessary.
```